docs/administration/vulnerability-scanning/_index.md

```diff
@@ -3,13 +3,25 @@ title: Vulnerability Scanning
 weight: 35
 ---
 
-Harbor provides static analysis of vulnerabilities in images through the open source [Clair](https://github.com/coreos/clair) project. 
+Harbor provides static analysis of vulnerabilities in images through the open source projects [Trivy](https://github.com/aquasecurity/trivy) and [Clair](https://github.com/coreos/clair). To be able to use Trivy, Clair or both you must have enabled Trivy, Clair or both when you installed your Harbor instance.
 
 {{< important >}}
-Clair is an optional component. To be able to use Clair you must have enabled Clair when you installed your Harbor instance.
+So far, Harbor supports only one system default scanner. The following principles will be applied to determine the system default scanner among the default installed scanners.
+
+For new installation case: 
+
+If only one scanner (either Trivy or Clair) is installed, the installed one will become the system default scanner automatically;
+
+If both Trivy and Clair are installed, Trivy will be the system default scanner then.
+
+For upgrading case:
+
+If there was an existing system default scanner in the previous version, no matter if it is Trivy or Clair, that scanner will be kept as system default scanner as it was;
+
```

Comment:
talk about the when installing with the different arguments, consider adding this, which leaves nothing to imagination 

Original Harbor had no scanner configured [this is the same case as for a brand new install]
i. Nothing. In this case no scanner is installed
ii. “with-trivy”. Trivy is installed and default
iii. “with-clair”. Clair is installed  and default
iv. “with-clair” and “with-trivy”. Both are installed. Trivy is the default

Original Harbor had a scanner ABC configured
i. Nothing. Error out since a scanner was configure pre-upgrade
ii. “with-trivy”. Trivy is installed. ABC scanner is default
iii. “with-clair”. Clair is installed . ABC scanner is default
iv. “with-clair” and “with-trivy”. Both are installed. ABC scanner is default

Reply:
I can append it as examples.

Reply:
add more statements and all the cases you mentioned above has been included.